sample-code/src/orchestration.ts

```diff
@@ -208,3 +208,43 @@ export async function orchestrationRequestConfig(): Promise<OrchestrationRespons
     }
   );
 }
+
+/**
+ * Ask about Generative AI Hub in SAP AI Core and ground the response.
+ * @returns The orchestration service response.
+ */
+export async function orchestrationGrounding(): Promise<OrchestrationResponse> {
+  const orchestrationClient = new OrchestrationClient({
+    llm,
+    templating: {
+      template: [
+        {
+          role: 'user',
+          content:
+            'UserQuestion: {{?groundingRequest}} Context: {{?groundingOutput}}'
+        }
+      ]
+    },
+    grounding: {
+      type: 'document_grounding_service',
+      config: {
+        filters: [
```

Comment:
Maybe we can consider some convenience API as a follow up for e.g., the filter.

Reply:
https://github.tools.sap/AI/gen-ai-hub-sdk-js-backlog/issues/186